scripts/check_doc_ipfs_hashes.py

```diff
@@ -244,11 +246,14 @@ def get_hash_by_attributes(
 
 
 def check_ipfs_hashes(  # pylint: disable=too-many-locals,too-many-statements
-    fix: bool = False,
+    paths: list[Path] = None, fix: bool = False
 ) -> None:
     """Fix ipfs hashes in the docs"""
 
-    all_md_files = Path("docs").rglob("*.md")
+    if paths is None:
+        paths = [Path("./docs")]
+
+    all_md_files = itertools.chain.from_iterable([path.rglob("*.md") for path in paths])
```

Comment:
I'm curious: why do we need this change? Wasnt the previous version getting all md files?

Reply:
This allows specifying additional folders if needed in other repositories.